platform_selector.py

```diff
@@ -0,0 +1,61 @@
+from selector import Selector
+from custom_dtypes import *
+# print(driver.platform_selected())
+
+# class Platform_Selector(Selector):
+class Platform_Selector(Selector):
+
+    def __init__(self, Repository):
```

Comment:
Better to follow lower snake case letters for method arguments like "Repository"